PoGo.NecroBot.CLI/Config/Translations/translation.id.json

```diff
@@ -270,7 +270,7 @@
     },
     {
       "Key": "gotUpToDateVersion",
-      "Value": "Sempurna! Kamu punya versi terbaru {0}"
+      "Value": "Info! Silahkan perbarui ke versi terbaru {0}"
```

Comment:
Hello, nice to meet you. ^^
In that line, I tried to make sentences emphasis to the user to always use the latest version of the bot.